cylc/flow/task_message.py

```diff
@@ -46,6 +46,24 @@
 
 STDERR_LEVELS = (getLevelName(level) for level in (WARNING, ERROR, CRITICAL))
 
+MESSAGE_MUTATION = '''
+mutation (
+  $wFlows: [WorkflowID]!,
+  $taskJob: String!,
+  $eventTime: String,
+  $messages: [[String]]
+) {
+  message (
+    workflows: $wFlows,
+    taskJob: $taskJob,
+    eventTime: $eventTime,
+    messages: $messages
+  ) {
+    result
+  }
+}
+'''
```

Comment:
> Would it be useful here too? Looks like Graphene provides objects to create mutations too: https://docs.graphene-python.org/en/latest/types/mutations/

Those are schema definition objects.

Reply:
Could be an idea to collect the strings into one place, but there's no (or almost no) duplication because each is tailored to the respective command (and I can't think of where else I'd use them).

Reply:
I wasn't sure if there were queries or fragments being shared among the commands. If there's little or no sharing, I think it's OK to leave as is. 90% of the commands required only the mutation string and that `pclient` call, so quite simple to change/maintain I think.